app/screens/transfer-target-input-screen/transfer-target-input-screen.tsx

```diff
@@ -111,9 +115,23 @@ export class TransferTargetInputScreen extends React.Component<TransferTargetInp
 
   _mapParamsToProps = (prepProps?: TransferTargetInputScreenProps) => {
     const prevAddress = prepProps && prepProps.navigation.getParam("address")
+    const likerId = this.props.navigation.getParam("likerId")
     const address = this.props.navigation.getParam("address")
-    if (!prevAddress && address) {
-      this.props.txStore.setTarget(address)
+    const memo = this.props.navigation.getParam("memo")
+    const amount = this.props.navigation.getParam("amount")
+    const skipToConfirm = this.props.navigation.getParam("skipToConfirm")
+    const target = likerId || address
+    if (!prevAddress && target) {
+      this.props.txStore.setTarget(target)
```

Comment:
Please use `setReceiver` instead of `setTarget` here to reset the `liker` in `TransferStore`